tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -1215,7 +1215,7 @@ public function test_wp_enqueue_code_editor_when_generated_array_by_compact_will
 	 * Testing `wp_enqueue_code_editor` with `array_merge`.
 	 *
 	 * @ticket 41871
-	 * @covers ::wp_enqueue_code_editor
+	 * @covers ::wp_enqueue_code_editor()
```

Comment:
_Without_ the parentheses is actually correct, let's undo these change here.

Reply:
I don't know how this happened, will revert.